pytripgui/plan_vc/plan_view.py

```diff
@@ -193,7 +207,9 @@ def select_opti_algorithm_view_to_this(self, opti_alg):
         ui.optiAlgorithm_comboBox.setCurrentIndex(index_of_opti_alg)
 
     def get_physical_dose_dist_state(self):
-        return self.ui.physicalDoseDist_checkBox.checkState()
+        if self.ui.physicalDoseDist_checkBox.checkState() == Qt.Checked:
```

Comment:
It does not make sense to mix boolean return type (True) with integer retirn type (0).
Either replace `0` with `False` or shorten the method to:
```
return self.ui.physicalDoseDist_checkBox.checkState() == Qt.Checked
```

Reply:
done